pkg/model/dev_test.go

```diff
@@ -298,8 +298,8 @@ func Test_LoadRemote(t *testing.T) {
 
 	dev.LoadRemote()
 
-	if dev.Command[0] != "/var/okteto/bin/remote" {
-		t.Errorf("remote command wasn't set: %s", dev.Command)
+	if dev.Command[0] != "uwsgi" {
```

Comment:
what is this for?

Reply:
test to validate that the command is not replaced when remote mode is enabled.